src/main/java/org/hl7/davinci/priorauth/ProcessClaimItemTask.java

```diff
@@ -142,4 +173,39 @@ private boolean process() {
 
         return ret;
     }
+
+    private Disposition sendAndGetDisposition(Bundle bundle, int seq, String address) throws IOException {
+
+        // create post request
+
+        OkHttpClient client = new OkHttpClient();
+        RequestBody body = RequestBody.create(null, bundle.toString());
+        Request request = new Request.Builder().url(address).post(body).build();
+        // if no response in alloted time return some default value
+        try (Response response = client.newCall(request).execute()) {
+            String answer = response.body().string();
+            return convertStringTDisposition(answer);
+        } catch (Exception e) {
+            return Disposition.CANCELLED;
```

Comment:
I don't think the default should be cancelled unless we're going to go through and actually cancel the claim. I think its fine for this method to just throw the exception since the calling method is catching it